homeassistant/components/qnap/strings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "[%key:component::qnap::config::step::user::title%]",
+        "description": "[%key:component::qnap::config::step::user::description%]",
+        "data": {
+          "host":"[%key:common::config_flow::data::host%]",
+          "username":"[%key:common::config_flow::data::username%]",
+          "password":"[%key:common::config_flow::data::password%]",
+          "port":"[%key:common::config_flow::data::port%]",
+          "ssl":"[%key:common::config_flow::data::ssl%]",
+          "verify_ssl":"[%key:common::config_flow::data::verify_ssl%]",
+          "timeout":"[%key:component::qnap::config::step::user::data::timeout%]",
+          "monitored_conditions":"[%key:component::qnap::config::step::user::data::monitored_conditions%]",
```

Comment:
Are these strings pointing back at themselves? I think their values are supposed to be in the strings.json and not only in en.json? That is how I understood it from the docs and how I see other components doing it.